Resources/Prototypes/Entities/Structures/Walls/meteor.yml

```diff
@@ -0,0 +1,221 @@
+# Define the meteor rock prototype separate from the asteroid rock to allow for different properties
```

Comment:
Im not sure what the comments mean since its already implemented..?

Reply:
I was just trying to explain what this does, though it could use some clarification

Reply:
It’s not really needed, if you want to clarify feel free to put a comment after the destructible component numbers explaining why the damage is as it is